etc/config-template.toml

```diff
@@ -462,8 +462,8 @@
 # enabled = false
 
 ## Maximum number of threads of `Titan` background gc jobs.
-# default: 1
-# max-background-gc = 1
+# default: 4
+# max-background-gc = 4
```

Comment:
What is the difference between `default` and `max-background-gc`?

Reply:
`default` is just a comment. `max-background-gc` is which user can un-comment and make change.